test/carbonite/transaction_test.exs

```diff
@@ -24,6 +24,11 @@ defmodule Carbonite.TransactionTest do
 
       assert get_field(changeset, :meta) == %{foo: 2, bar: 1}
     end
+
+    test "uses the default carbonite_prefix" do
```

Comment:
in the suite above you've added the test to a `describe "Schema"` section, IMO that makes more sense than in the `changeset/1` section